typescript/src/lib/moderation/structures/ModerationMessageEvent.ts

```diff
@@ -40,13 +39,14 @@ export abstract class ModerationMessageEvent<T = unknown> extends Event {
 		const preProcessed = await this.preProcess(message);
 		if (preProcessed === null) return;
 
-		const [filter, adder, language] = await message.guild.readSettings((settings) => [
+		const [channelId, filter, adder, language] = await message.guild.readSettings((settings) => [
```

Comment:
```suggestion
		const [logChannelId, filter, adder, language] = await message.guild.readSettings((settings) => [
```

And update references